interpreter/exec/simd.ml

```diff
@@ -200,12 +201,18 @@ struct
       val to_shape : Rep.t -> Int.t list
       val of_shape : Int.t list -> Rep.t
       val num_lanes : int
+      (* For I8 and I16, since they are stored signed in an int32, this mask
+       * is used for unsigned extracts, to get the right unsinged value. *)
+      val unsigned_mask : Int.t
```

Comment:
I'm not sure I understand why this needs to leak here. Shouldn't that be encapsulated in the Int8/Int16 modules?

Reply:
Move this into Int.ml, created an `as_unsigned` method there.